nixos/modules/services/misc/klipper.nix

```diff
@@ -35,6 +35,30 @@ in
         description = lib.mdDoc "Path of the API socket to create.";
       };
 
+      configPath = mkOption {
+        type = types.nullOr types.path;
+        default = "/etc/klipper.cfg";
+        example = "/var/lib/klipper/printer.cfg";
+        description = lib.mdDoc "Path to Klipper config file.";
+      };
```

Comment:
Just gate the default on `stateVersion` and add the info on the release notes.

Reply:
ty, good idea)